### DISCUSSION TITLE
Flashcard tags in YAML frontmatter

### DISCUSSION THREAD

**Discussion by Masstronaut** [st3v3nmw/obsidian-spaced-repetition#160] [Q&A] [answered]
Is there a way to use the YAML frontmatter to indicate that a note contains flashcards? I'd like to keep metadata (ie that a note contains flashcards) out of the note content. I've tried putting my `#flashcards` in the frontmatter but the cards aren't appearing in my review queue.

**Comment by st3v3nmw** [comment, marked as answer]
How about `tags: [flashcards]`? That should do it.

**Reply by Masstronaut** [reply]
Perfect! Nested tags (`flashcards/deck`) seem to work too. Thanks!
